### PR TITLE
[v7r0] Add documentation about how to install WMS: admin tutorial

### PR DESCRIPTION
**Pull request by aldbr**
I made a documentation on how to install the WMS a long time ago but forgot to push it.
I've just tried it again with dirac v7r0 and it works.

BEGINRELEASENOTES
*docs
NEW: admin tutorial on how to install the WorkloadManagementSystem
ENDRELEASENOTES

### CODE REVIEW

**Review by fstagni — APPROVED** [review]

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+  agent WorkloadManagement_JobCleaningAgent is installed, runit status: Run
+  [dirac-tuto]> install agent WorkloadManagement PilotStatusAgent
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/PilotStatusAgent
+  agent WorkloadManagement_PilotStatusAgent is installed, runit status: Run
+  [dirac-tuto]> install agent WorkloadManagement StalledJobAgent
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/StalledJobAgent
+  agent WorkloadManagement_StalledJobAgent is installed, runit status: Run
+  [dirac-tuto]> install executor WorkloadManagement JobPath
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Tutorials/installWMS.rst:126]: This isn't how the executors are installed in certification (and LHCbDirac?), where there is just one executor called "Optimizers" that calls the executors defined in the "Load" parameter.

Not sure what or if anything is better?

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+Due to installation issue, the synchronization CS-Pilot is disabled by default, modify the ``/Operations/Defaults/Pilot`` section in the ``/opt/dirac/etc/dirac.cfg`` file as ``dirac``::
+
+    UpdatePilotCStoJSONFile = True
+
+Still as ``dirac``, create the pilot repository that will contain all the pilot files that will be updated whenever a CS update is triggered::
+
+    mkdir -p /opt/dirac/webRoot/www/pilot
+
+
+Note: Do not put the Pilot configuration in ``Operation/Defaults``, DIRAC would not be able to get it.
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Tutorials/installWMS.rst:289]: Why is that?

And
```suggestion
.. warning:: Do not put the Pilot configuration in ``Operation/Defaults``, DIRAC would not be able to get it.
```

**Review by andresailer — COMMENTED** [review]
Thank you for this, looks very nice.
I just a a few comments and questions.

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+  executor WorkloadManagement_JobSanity is installed, runit status: Run
+  [dirac-tuto]> install executor WorkloadManagement InputData
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/InputData
+  executor WorkloadManagement_InputData is installed, runit status: Run
+  [dirac-tuto]> install executor WorkloadManagement JobScheduling
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/JobScheduling
+  executor WorkloadManagement_JobScheduling is installed, runit status: Run
+  [dirac-tuto]> restart WorkloadManagement *
```
[the remark this answers is not in the document]
Comment (fstagni) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst:142]: Yes, that's also needed.

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst (hunk 1 above)
Comment (fstagni) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst:126]: You probably want one "optimizer" running all the executors.

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst (hunk 2 above)
Comment (aldbr) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst]: I didn't remember so I checked again, and this is very strange.
When I put the `Pilot` configuration into `Operation/Defaults`, then the Site Director is not able to fetch parameters to built the Pilot options.
If I set a breakpoint, it is able to get them. 
Is there any known issue with the Operation helper?

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst (hunk 1 above)
Comment (aldbr) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst]: Alright, thanks I didn't know about that

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (aldbr) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst:142]: Yes you're right, I've added it

File: docs/source/AdministratorGuide/Tutorials/installWMS.rst (hunk 2 above)
Comment (andresailer) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst]: All this should be `Operations`, with `s`! But that isn't the problem, is it?

Comment (aldbr) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst]: Oops, forgot the `s` multiple times, I added it everywhere, thanks.
No, this is not the problem...

Comment (fstagni) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst]: I would be surprised if that's (still) valid. Did you try with a recent version?

Comment (aldbr) [reply docs/source/AdministratorGuide/Tutorials/installWMS.rst]: I tried with v7r0p36

**Review by fstagni — APPROVED** [review]

**Pull request merged**